assets/css/main.scss

```diff
@@ -52,148 +52,167 @@ body.dark .jumbotron {
-  color: #d5d5d5;
-}
-body.dark .dropdown-item:hover, .dropdown-item:focus {
-  background-color: #1a1a1a;
-}
-
-body.dark .list-group-item[data-v-1084c69e]:hover:not(.active):not(.h5) {
-  background-color: #d5d5d5;
-}
-
-body.dark .modal-content {
-  background-color: #252525;
-  color: #d5d5d5;
-}
-
-body.dark .nav-item:first-child .nav-link,
-.nav-item:first-child button {
-  border-left-color: #242424;
-}
-
-body.dark .nav-item .nav-link,
-.nav-item button {
-  border-right-color: #242424;
-}
-
-body.dark .form-control {
-  border-color: #181818;
-  color: #94a3b1;
-}
-
-body.dark .form-control:focus {
-  color: #eee;
-}
-
-body.dark .heading {
-  border-bottom-color: #181818;
-}
-
-body.dark .dropdown-divider,
-body.dark .dropdown-menu {
-  border-color: #242424;
-}
-
-body.dark .list-group-item-action {
-  color: #616a74;
-}
-
-body.dark .list-group-item.active {
-  border-color: #232222;
-  color: #fff;
-}
-
-body.dark .list-group-item {
-  border-top-color: #232222;
-}
-
-/* post text */
-body.dark .media-body {
-  color: #bdc5cf;
-}
-body.dark .text-muted {
-  color: #8e9aa5 !important;
-}
-
-/* usernames */
-body.dark .text-gray-dark {
-  color: #ddd !important;
-}
-
-/* new post line */
-body.dark .modal-header {
-  border-bottom-color: #232323;
-}
-
-/* focused post */
-body.dark .list-group-item-warning.list-group-item-action:hover,
-body.dark .list-group-item-warning.list-group-item-action:focus,
-body.dark .list-group-item-warning {
-  background-color: #1d2324;
-}
-
-/* top menu toggle */
-body.dark .navbar-light .navbar-nav .show > .nav-link,
-.navbar-light .navbar-nav .active > .nav-link,
-.navbar-light .navbar-nav .nav-link.show,
-.navbar-light .navbar-nav .nav-link.active {
-  color: rgb(189, 148, 139);
-}
-
-/* beta text */
-body.dark .navbar-light .navbar-brand:hover,
-.navbar-light .navbar-brand:focus {
-  color: #fff;
-}
-
-/* file list table */
-body.dark .table th,
-body.dark .table td {
-  border-color: #232323;
-}
-
-/* side panel hover */
-body.dark .list-group-item:hover:not(.active):not(.h5) {
-  background-color: #2a2a2a !important;
-  color: #ddd;
-}
-
-/* search form */
-body.dark .search-form input[type='search']:focus {
-  color: #262626;
-}
-
-/* messages */
-body.dark .balloon {
-  background-color: #202020;
-}
-body.dark .balloon.other::before {
-  border-right-color: #202020;
-}
-body.dark .balloon.me::before {
-  border-left-color: #202020;
+body.dark {
+  .navbar {
+    background-color: #252525;
+    border-bottom-color: #212121;
+  }
+
+  .card {
+    background-color: #252525;
+  }
+
+  .list-group-item {
+    background-color: #252525;
+  }
+
+  .form-control {
+    background-color: #252525;
+  }
+
+  .dropdown-menu {
+    background-color: #252525;
+    color: #d5d5d5;
+  }
+
+  .dropdown-item {
+    color: #d5d5d5;
+  }
+  .dropdown-item:hover, .dropdown-item:focus {
+    background-color: #1a1a1a;
+  }
+
+  .list-group-item[data-v-1084c69e]:hover:not(.active):not(.h5) {
```

Comment:
Umm... I don't understand this class that which components. Can you write as scoped css in a vue component?

Reply:
Removed offending code. I think it wasn't necessary anymore.